ts/parts/Options.ts

```diff
@@ -3675,6 +3676,16 @@ H.defaultOptions = {
          * @apioption tooltip.split
          */
 
+        /**
+         * Prevents the tooltip from switching or closing, when touched or
+         * pointed.
+         *
+         * @type      {boolean}
+         * @since     8.0.1
+         * @product   highcharts highstock
+         * @apioption tooltip.sticky
```

Comment:
Is `sticky` a descriptive name? I think something like `stickOnHover` better describes the feature.

Reply:
This should be solved.